app/assets/javascripts/Components/Modals/create_modify_annotation_panel_modal.jsx

```diff
@@ -199,108 +193,116 @@ class CreateModifyAnnotationPanel extends React.Component {
         onRequestClose={this.props.onRequestClose}
         parentSelector={() => document.querySelector("#content")}
       >
-        <div
-          id="annotation-modal"
-          style={{
-            width: "600px",
-            display: "block",
-          }}
-        >
-          <h2>{this.props.title}</h2>
-          <form onSubmit={this.onSubmit}>
-            <div className={"modal-container-vertical"}>
-              <div>
-                <label>
-                  <textarea
-                    required={true}
-                    id="new_annotation_content"
-                    name="content"
-                    placeholder={I18n.t("results.annotation.placeholder")}
-                    value={this.state.content}
-                    onChange={this.handleChange}
-                    rows="8"
-                    autoFocus={true}
-                  />
-                </label>
-
-                <div className={this.state.show_autocomplete ? "" : "hidden"}>
-                  <ul className="tags" key="annotation_completion" id="annotation_completion">
-                    <li className="annotation_category" id="annotation_completion_li">
-                      <p id="annotation_completion_text"></p>
-                      <div>
-                        <ul id="annotation_text_list"></ul>
-                      </div>
-                    </li>
-                  </ul>
-                </div>
-
-                <h3>{I18n.t("preview")}</h3>
-
-                <input
-                  type="hidden"
-                  id="annotation_text_id"
-                  name="annotation_text_id"
-                  value={this.state.annotation_text_id}
-                />
-
-                <div id="annotation_preview" className="preview"></div>
-                {this.props.is_reviewer ? (
-                  <input
-                    type="hidden"
-                    id="new_annotation_category"
-                    name="category_id"
-                    value={I18n.t("annotation_categories.one_time_only")}
-                  />
-                ) : (
-                  <div className="inline-labels">
-                    <label htmlFor="new_annotation_category">
-                      {I18n.t("activerecord.models.annotation_category.one")}
+        <Tabs disableUpDownKeys>
```

Comment:
The reason I commented on this line in particular is that I wanted you to move it. So to be clear, you should move `Tabs` so that it's inside the `div` within the `form` itself.